airbyte-workers/src/main/java/io/airbyte/workers/temporal/CancellationHandler.java

```diff
@@ -64,7 +69,7 @@ public void checkAndHandleCancellation(Runnable onCancellationCallback) throws W
         context.heartbeat(null);
       } catch (ActivityCompletionException e) {
         onCancellationCallback.run();
-        throw new WorkerException("Worker cleaned up after exception", e);
+        LOGGER.warn("Job either timeout-ed or was cancelled.");
```

Comment:
instead of throwing a noisy exception, a log line here felt more useful to me. this also simplifies the interface by letting us get rid of the `throws`